CRM/Civicase/Setup/CreateSafeFileExtentionOptionValue.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Class CRM_Civicase_Setup_CreateSafeFileExtentionOptionValue.
+ */
+class CRM_Civicase_Setup_CreateSafeFileExtentionOptionValue {
```

Comment:
The naming here is wrong. Should be `Extension` not `Extention`so correct class name should be `CRM_Civicase_Setup_CreateSafeFileExtensionOptionValue`